src/runtime_src/core/edge/drm/zocl/edge/zocl_edge_xclbin.c

```diff
@@ -403,7 +407,7 @@ zocl_xclbin_read_axlf(struct drm_zocl_dev *zdev, struct drm_zocl_axlf *axlf_obj,
 	zocl_init_mem(zdev, slot);
 
 	/* Createing AIE Partition */
-	zocl_create_aie(zdev, axlf, aie_res, hw_gen);
+	zocl_create_aie(zdev, axlf,xclbin, aie_res, aie_res_bin, hw_gen, part_id);
```

Comment:
better to extract all of this inside function instead of passing them as arguments